Holovibes/includes/gui/windows/MainWindow.hh

```diff
@@ -205,6 +210,7 @@ class MainWindow : public QMainWindow
     Ui::MainWindow* ui_;
     std::vector<Panel*> panels_;
     std::shared_ptr<LightUI> light_ui_;
+    api::Api& api_;
```

Comment:
Why here ? (legit question)

Reply:
Just to speed up the access to API function